spec/tapioca/cli_spec.rb

```diff
@@ -569,6 +575,107 @@ def body=(body); end
       refute_path_exists("#{outdir}/namespace/comment.rbi")
       refute_path_exists("#{outdir}/user.rbi")
     end
+
+    describe("verify") do
+      describe("with no changes") do
+        before do
+          execute("dsl")
+        end
+
+        it 'does nothing if no RBIs have changed' do
+          output = execute("dsl", "--verify")
+
+          assert_includes(output, <<~OUTPUT)
+            Nothing to do, all RBIs are up-to-date.
+          OUTPUT
+        end
+
+        it ' returns the correct exit code' do
```

Comment:
I think these `returns correct exit code` tests be combined with the tests just above them, I'd personally prefer it since it makes the file shorter and tad bit easier to read, but it's your call. No strong opinions.